.github/workflows/test-win-exe-w-embed-py.yaml

```
@@ -1,203 +1,212 @@
+              # Logo of app
+              cp assets/openms.ico SourceDir
+
+         - name: Generate WiX XML file
+           shell: bash
+           run: |
+              cat <<EOF > streamlit_exe.wxs
+              <?xml version="1.0"?>
+              <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+                <Product Id="*" Name="${{ env.APP_NAME }}" Language="1033" Version="${{ env.APP_VERSION }}.0" Codepage="1252" Manufacturer="OpenMS Developer Team" UpgradeCode="${{ env.APP_UpgradeCode }}">
+                  <MajorUpgrade 
+<<<<<<< Updated upstream
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 122: could not find expected ':'

(syntax-check)